tests/integration/test_vulnerability_detector/test_providers/test_enabled.py

```diff
@@ -0,0 +1,102 @@
+import os
+import pytest
+
+from wazuh_testing import global_parameters
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.vulnerability_detector import make_callback, callback_detect_no_feeds_downloaded
+
+# Marks
+pytestmark = pytest.mark.tier(level=1)
```

Comment:
I would consider this kind of test as tier 0 since it is basic functionality.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/689/commits/231750a45f80e84dd90c5506f0b7c3f41b64b775